### ISSUE TITLE
Widgets 

### ISSUE THREAD

**Issue by jerryop** [vfsfitvnm/ViMusic#234]
I love this app but this app doesn't have any widget like other music apps have can we get a widget in next update btw app is very good 💗

**Comment by arfanmrafeek** [comment]
Yes, We need widget in Material You UI as possible. We love to have it.

**Comment by vfsfitvnm** [comment]
Jetpack compose cannot be used to build widgets, so we can only use xml.
Adding a single widget increases the apk file size of 600kB, so I'm not sure if it's more appropriate to put widgets in a separate apk

**Comment by ARBoyGo** [comment]
> separate apk

It's better. Separate apk for widgets.

**Comment by ARBoyGo** [comment]
> so I'm not sure if it's more appropriate to put widgets in a separate apk

Making separate is good cause some might not widgets. And apk size now it's better. So let's get separate apk for those who want widget more logical.

**Comment by TigerbyteDev** [comment]
Are there some widgets yet?

**Comment by DevNegm** [comment]
Any updates?

**Comment by hasa67** [comment]
This is s great idea. I prefer a widget too. Please let me know if you make one for the app.

**Comment by lazarr16** [comment]
I love Vimusic and have been using it for months.A widget would be an amazing addition to the experience.

**Comment by andietablante18** [comment]
> > so I'm not sure if it's more appropriate to put widgets in a separate apk
> 
> Making separate is good cause some might not use/like widgets. And apk size now it's better. So let's get separate apk for those who want widget more logical. 

I definitely agree on this one. Such a great idea. 

Please notice, @vfsfitvnm

**Comment by ashish-aa** [comment]
Thread is old as 2yrs now and there's still no update on this..
